data/s3dis/README.md

```diff
@@ -0,0 +1,56 @@
+### Prepare S3DIS Data
+We follow the procedure in [pointnet](https://github.com/charlesq34/pointnet).
```

Comment:
There should be an empty line above the paragraph?

Reply:
Fix.